DuckDuckGo/ContentBlockerViewController.swift

```diff
@@ -27,8 +27,8 @@ class ContentBlockerViewController: UITableViewController {
     @IBOutlet weak var httpsLabel: UILabel!
     @IBOutlet weak var blockCountCircle: UIImageView!
     @IBOutlet weak var blockCount: UILabel!
-    @IBOutlet weak var blockingEnabledToggle: UISwitch!
     @IBOutlet weak var blockThisDomainToggle: UISwitch!
+    @IBOutlet weak var tutorialText: UILabel!
```

Comment:
Good call, this is a UITableView so I've used a custom UITableViewCell class . I sat on the fence about this as it's more code to get reuse in than just repeating it, but I think the codebase is better for it - repetition is a maintenance nightmare.